.pre-commit-config.yaml

```diff
@@ -78,7 +78,7 @@ repos:
   - id: mypy
     name: mypy
     entry: mypy
-    args: [--config-file=pyproject.toml]
+    args: [--config-file=pyproject.toml, --pretty]
```

Comment:
I didn't know about the `--pretty` option before, but it results in much friendlier error messages!